lib/game.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'board.rb'
+require_relative 'players/human.rb'
+require_relative 'players/computer.rb'
+#require_relative 'player.rb'
+require 'pry'
+
+class Game
+	attr_accessor :board, :player_1, :player_2, :turn_res
+	WIN_COMBINATIONS = [
+		[0,1,2],
+		[0,3,6],
+		[0,4,8],
+		[1,4,7],
+		[2,4,6],
+		[3,4,5],
+		[6,7,8],
+		[2,5,8]
+		]
+	
+	def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+	end	
+	
+	def current_player
+		self.board.cells.count("O") == self.board.cells.count("X") ? self.player_1 : self.player_2
+	end
+	
+	def won?
+		!WIN_COMBINATIONS.detect { |x| self.board.cells[x[0]] == self.board.cells[x[1]] && self.board.cells[x[0]] == self.board.cells[x[2]] && self.board.cells[x[0]] != " "}.nil?
+	end  
+	
+	def draw?
+		self.board.full? && !self.won?
+	end
+	
+	def over?
+		self.won? || self.draw?
+	end
+	
+	def winner
+		if self.won? && !WIN_COMBINATIONS.detect { |x| self.board.cells[x[0]] == self.board.cells[x[1]] && self.board.cells[x[0]] == self.board.cells[x[2]] && self.board.cells[x[0]] == "X"}.nil?
+			return "X"
+		elsif self.won?
+			return "O"
+		else 
+			return nil
+		end

```

Comment:
As written, this method iterates through `WIN_COMBINATIONS` three times - for `!WIN_COMBINATIONS.detect` in this method, and the two calls to `self.won?`. I bet you could find a way to simplify this.